components/slides/HomePage.js

```diff
@@ -56,24 +56,29 @@ export default function HomePage() {
                 Profile Builder
               </p>
               {/* Text Input */}
-              <div className="2xl:my10 mt-8 flex flex-col items-center sm:flex-row md:my-8">
+              <div>
                 <form className="w-full" onSubmit={onNext}>
-                  <input
-                    type="text"
-                    value={input}
-                    required={true}
-                    onChange={(e) => setInput(e.target.value)}
-                    autoFocus={true}
-                    className="focus:border-border-[#ECA227] inline w-full border-b-2 border-[#ECA227] bg-transparent text-xl outline-none focus:border-b-4 sm:w-11/12 sm:text-3xl md:w-10/12 md:text-xl lg:w-8/12 lg:text-2xl 2xl:text-3xl"
-                    placeholder="Enter Your GitHub Username"
-                  />
-                  <button type="Submit">
-                    <RIGHT_ARROW_SVG />
-                  </button>
+                  <div className="mt-8 flex flex-col items-center sm:flex-row md:my-8 2xl:my-10">
+                    <input
+                      type="text"
+                      value={input}
+                      required={true}
+                      onChange={(e) => setInput(e.target.value)}
+                      autoFocus={true}
+                      className="focus:border-border-[#ECA227] inline w-full border-b-2 border-[#ECA227] bg-transparent p-2 py-3 text-lg outline-none transition-all duration-75 focus:rounded-full focus:border-2 focus:px-4 sm:w-11/12 sm:text-3xl md:w-10/12 md:text-xl lg:w-8/12 lg:text-xl 2xl:text-3xl"
```

Comment:
Making this rounded on focus shifts the placeholder text. Which looks bad. Either keep it rounded all the time or try something else.